### PR TITLE
Removed required attribute from TLS certificate

### PR DESCRIPTION
**Pull request by ManuelB**

### CODE REVIEW

File: src/renderer/modules/settings/screens/formBuilder.ts [hunk 1]
```diff
@@ -312,7 +312,7 @@ export function getFormSections(repositoryData: TRepositoryData): IConfigSection
           type: 'file-path',
           hide: repositoryData[TLS_AUTH_TYPE_CONFIG] !== TlsAuthType.ServerClientCertAuth_Pfx,
           infoText: translate('info_text_pfx_file'),
-          required: repositoryData[TLS_AUTH_TYPE_CONFIG] === TlsAuthType.ServerClientCertAuth_Pfx,
+          required: false, // repositoryData[TLS_AUTH_TYPE_CONFIG] === TlsAuthType.ServerClientCertAuth_Pfx,
```
Comment (serdarde) [inline src/renderer/modules/settings/screens/formBuilder.ts:315]: Hi Manuel, thank you for the MR.

Actually, the logic we have implemented says that it is required if you choose the p12 certificate for authentication, and we think that this is the right way.

If you select p12 file authentication, then you really need to upload a p12 file.

**Comment by ManuelB** [comment]
@serdarde I can't save the dialog. I opened an issue here: https://github.com/gematik/app-Authenticator/issues/27

**Comment by ManuelB** [comment]
I am using an Ubuntu Linux 20.04 (Linux manuel-XPS-13-9305 5.15.0-92-generic #102~20.04.1-Ubuntu SMP Mon Jan 15 13:09:14 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux) and I am used the v4.8.1 branch and started the system with npm run dev.

**Comment by serdarde** [comment]
Hi Manuel,

Sorry for the late response.

As I mentioned before, the issue is related to selecting the certificate in the TLS Authentication option.

If you had selected “No Authentication” there, you wouldn’t have encountered this issue.

The screenshot you shared shows that the warning appears in the wrong place, and we have created a ticket for this. If there are no further issues, we would like to close this Merge Request.

**Pull request closed without merging**